package.json

```diff
@@ -5,6 +5,7 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "scripts": {
+    "install": "npm run build",
```

Comment:
Not this line, but in `package.json`, I think we should bump `resin-bundle-resolve` to 4.1.2 to pickup this PR:
https://github.com/balena-io-modules/resin-bundle-resolve/pull/38
... seeing that `resin-builder`'s own package.json does not mention `resin-bundle-resolve`.

Reply:
It would have been picked up regardless but I've made it explicit now